charts/s1-agent/templates/common/secrets.yaml

```diff
@@ -11,5 +11,19 @@ data:
 {{- if not (mustRegexMatch "^\\{\"url\".+\\}$" $decode) }}
 {{ fail "The site_key.value does not look like a valid site/group token. You must use a site token/group token exported from your console (under the desired site/group scope => Site Info/Group Info. Make sure to copy the full string and not omit any trailing '=' characters." }}
 {{- end -}}
-{{- end -}}
+{{- end }}
+
+---
 
+{{- if include "helper.secret.create" . }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "helper.secret.name" . }}
+  labels: {{- include "sentinelone.helper.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+type: kubernetes.io/tls
+data: {{- include "helper.certificates" . | nindent 2 }}
+{{- end -}}
```

Comment:
Shouldn't this end with no `-` if the if block started with on `-`?

Reply:
it doesn't matter as much as I know. The '-' at the end means that it is the end of the section and adding a new section after that, will cause an error unless the last '-' is being removed.